docs/_pages/general/rooting-instructions.md

```diff
@@ -7,6 +7,114 @@ order: 9
 
 This page contains an incomplete overview of installation instructions for various robots
 
+## Dreame
+
+Dreame rooting is currently possible for
+
+* Dreame L10 Pro
+* Dreame Z10 Pro
+* Dreame F9
+* Dreame D9
+* Xiaomi Vacuum Robot 1C
+* Xiaomi Vacuum Robot 1T
+
+It has been released with Dennis Giese's DEF CON 29 Talk [Robots with lasers and cameras but no security Liberating your vacuum](https://youtu.be/EWqFxQpRbv8?t=1525).
+For more information, head over to the [Dustbuilder](https://builder.dontvacuum.me/).
+
+It is also recommended to join the [Dreame Robot Vacuum Telegram Usergroup](https://t.me/joinchat/VwEy4UeBrf45MTZi) for rooting support etc.
+
+### Reset-Button UART
+
+**Dreame is aware of this and might patch it in newer firmwares. Therefore, don't update your robot in the Mi Home App (root works on .1073 but .1105 patches this method)**
```

Comment:
1073 and 1105 are version names specific to the robot model (L10 in this case) which is why we can't have that here